src/cleanvision/issue_managers/duplicate_issue_manager.py

```diff
@@ -10,6 +10,41 @@
 from cleanvision.utils.base_issue_manager import IssueManager
 from cleanvision.utils.constants import SETS, DUPLICATE
 
+import multiprocessing
+
+# psutil is a package used to count physical cores for multiprocessing
+# This package is not necessary, because we can always fall back to logical cores as the default
+try:
+    import psutil
+
+    psutil_exists = True
+except ImportError as e:  # pragma: no cover
+    psutil_exists = False
+
+
+def get_hash(image: Image, params: Dict[str, Any]) -> str:
```

Comment:
Since this is required for multiprocessing, remove the duplicate method in class and use this method everywhere